### DISCUSSION TITLE
How do I contribute to the docs?

### DISCUSSION THREAD

**Discussion by BunningsWarehouseOfficial** [falcosecurity/falco-talon#502] [Q&A] [answered]
I can't figure out the best way to suggest changes to the docs. Links I've tried in the docs pages themselves lead to 404 pages on GitHub. The hugo-docs branch seems out of date.

**Comment by Issif** [comment, marked as answer]
Hi,

The code base of Talon project itself have been transferred to the falcosecurity organization recently, this explains why the links for the edit are broken now, will try to fix that. Thanks for the notice.

The sources of the docs are here: https://github.com/falco-talon/falco-talon.github.io, all contributions are welcome.
